tasks/yum.yml

```diff
@@ -22,7 +29,15 @@
     gpgcheck: "{{ rke2_repo_common_gpg_check }}"
     gpgkey: "{{ rke2_repo_common_gpg_key }}"
     enabled: yes
+  when: not stat_rke2_common.stat.exists
+
+# Does the Rancher RKE2 1.19 repo exist already
+- name: Check to see if rke2-v1.19.repo exists
+  stat:
+    path: '/etc/yum.repos.d/rke2-v1.19.repo'
+  register: stat_rke2_v119
 
+# Add RKE2 1.19 repo if it doesn't exist
 - name: add the rke2-v1.19 repo
   yum_repository:
     name: rke2-v1.19
```

Comment:
This works for the current release of RKE2, but ideally we'd pass this in as a user-supplied var

Reply:
Excellent point. Made the change.